src/main/java/uk/ac/ebi/ena/webin/cli/service/exception/ServiceiException.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2018-2021 EMBL - European Bioinformatics Institute
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package uk.ac.ebi.ena.webin.cli.service.exception;
+
+public class
+ServiceiException extends RuntimeException

```

Comment:
- The exception name is misspelled. Think about the exceptions and how we should preserve the current error handling/reporting in Webin-CLI when we move its code from there to here.